<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTask</class>
 <widget class="QWidget" name="addTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/rec/Picture/list.png</normaloff>:/rec/Picture/list.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ebeef7;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>365</y>
     <width>688</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:35px;
background-color:#423d3c;
color:white;s</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введіть своє завдання</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeTaskInDatabase">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>364</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff; /* цвет текста */
  text-decoration: none; /* убирать подчёркивание у ссылок */
  user-select: none; /* убирать выделение текста */
  background: rgb(212,75,56); /* фон кнопки */
 
  outline: none; /* убирать контур в Mozilla */
font-weight:900;</string>
   </property>
   <property name="text">
    <string>Добавити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTaskButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">display: inline-block;
  color: white;
  font-weight: 700;
  text-decoration: none;
  user-select: none;
  outline: none;
  border: 2px solid;
  border-radius: 1px;
  transition: 0.2s;
border-radius:10px;
background-color:#b8b8b8;
font-size:15px;</string>
   </property>
   <property name="text">
    <string>Показати завдання</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
font-weight:900;
font-family: Georgia, serif;
font-style: italic;
background-color:#423d3c;
color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToMenu">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">display: inline-block;
  color: white;
  font-weight: 700;
  text-decoration: none;
  user-select: none;
  outline: none;
  border: 2px solid;
  border-radius: 1px;
  transition: 0.2s;
font-size:15px;
border-radius:10px;
background-color:#b8b8b8;</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRet">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
